README.md

```diff
@@ -376,6 +377,12 @@ Z = sqrt(tag<1>(X) * tag<1>(X) + tag<2>(Y) * tag<2>(Y));
 Here, the generated kernel will have one parameter for each of the vectors `X`
 and `Y`.
 
+### <a name="pointer-qualifiers"></a>pointer qualifiers
+VexCL has implmented some of the pointer qualifiers for kernel input variables.
+The programmer can now tag non ailising pointer with the 'restricted', this adds the 'restrict' keyword to the pointer,
+or and constant arrays 'restricted_const' which results is 'const type * restrict'.
+
```

Comment:
There are couple of typos here. (implmented, ailising).
I would rephrase it anyway, may be to
```
VexCL allows the user to qualify the pointers passed to OpenCL/CUDA kernels with 'restrict' keyword, which limits the effects of pointer aliasing, aiding optimizations. This is achieved by wrapping a vector in an expression with either `restricted()` or `const_restricted()` function.
```